magicctapipe/scripts/mars/tests/labels_and_units_for_params_comparison.py

```diff
@@ -0,0 +1,145 @@
+import math
+
+mars_params = {
+    "event_id" : "MRawEvtHeader_1.fStereoEvtNumber",
+    "length_M1" : "MHillas_1.fLength", "length_M2" : "MHillas_2.fLength",
+    "width_M1" : "MHillas_1.fWidth", "width_M2" : "MHillas_2.fWidth",
+    "size_M1" : "MHillas_1.fSize", "size_M2" : "MHillas_2.fSize",
+    "slope_M1" : "MHillasTimeFit_1.fP1Grad", "slope_M2" : "MHillasTimeFit_2.fP1Grad",
+    "delta_M1" : "MHillas_1.fDelta", "delta_M2" : "MHillas_2.fDelta",
+    "cogx_M1" : "MHillas_1.fMeanX", "cogx_M2" : "MHillas_2.fMeanX",
+    "cogy_M1" : "MHillas_1.fMeanY", "cogy_M2" : "MHillas_2.fMeanY",
+    "hmax" : "MStereoPar.fMaxHeight",
+    "corex" : "MStereoPar.fCoreX",
+    "corey" : "MStereoPar.fCoreY",
+    "az" : "MStereoPar.fDirectionAz",
+    "zd" : "MStereoPar.fDirectionZd",
+    "impact_M1" : "MStereoPar.fM1Impact", "impact_M2" : "MStereoPar.fM2Impact",
+    "hmax_disp" : "MStereoParDisp.fMaxHeight",
+    "corex_disp" : "MStereoParDisp.fCoreX",
+    "corey_disp" : "MStereoParDisp.fCoreY",
+    "az_disp" : "MStereoParDisp.fDirectionAz",
+    "zd_disp" : "MStereoParDisp.fDirectionZd",
+    "impact_disp_M1" : "MStereoParDisp.fM1Impact", "impact_disp_M2" : "MStereoParDisp.fM2Impact",
+    # "n_islands" : "MImagePar_1./MImagePar_1.fNumIslands",
+    "leakage_M1" : "MNewImagePar_1./MNewImagePar_1.fLeakage1", "leakage_M2" : "MNewImagePar_2./MNewImagePar_2.fLeakage1"
+
+}
+mcp_params = {
+    "length_M1" : "length", "length_M2" : "length",
+    "width_M1" : "width", "width_M2" : "width",
+    "size_M1" : "intensity", "size_M2" : "intensity",
+    "slope_M1" : "slope", "slope_M2" : "slope",
+    "delta_M1" : "psi", "delta_M2" : "psi",
+    "cogx_M1" : "y", "cogx_M2" : "y",
```

Comment:
I think it would be good to add here a comment about the difference of the camera frame definition between MARS and ctapipe